### PR TITLE
add marriage configuration on/off for vital event

### PR DESCRIPTION
**Pull request by kayumuzzaman**

### CODE REVIEW

File: packages/migration/migrations/application-config/20230208081755-add-marriage-configuration.js [hunk 1]
```diff
@@ -42,7 +45,10 @@ export const down = async (db, client) => {
     await session.withTransaction(async () => {
       await db
         .collection('configs')
-        .updateMany({}, { $unset: { MARRIAGE: '' } })
+        .updateMany(
+          {},
+          { $unset: { MARRIAGE: '', MARRIAGE_REGISTRATION: false } }
```
Comment (Zangetsu101) [inline packages/migration/migrations/application-config/20230208081755-add-marriage-configuration.js:50]:
```suggestion
          { $unset: { MARRIAGE: '', MARRIAGE_REGISTRATION: '' } }
```
Although it doesn't matter what we use as the value but using "false" might be a bit confusing.

**Comment by Zangetsu101** [comment]
@euanmillar We want marriage to be disabled by default right? If so @kayumuzzaman the value should be "false" in the migration

**Comment by euanmillar** [comment]
@kayumuzzaman @Zangetsu101 I agree.  In the migration, the value should be false.

File: packages/migration/migrations/application-config/20230208081755-add-marriage-configuration.js (hunk 1 above)
Comment (kayumuzzaman) [reply packages/migration/migrations/application-config/20230208081755-add-marriage-configuration.js]: changed the default value to false

**Review by Zangetsu101 — APPROVED** [review]

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**